templates/default/openssh.conf.erb

```diff
@@ -117,3 +117,8 @@ UseRoaming <%= @node['ssh-hardening']['ssh']['client']['roaming'] ? 'yes' : 'no'
 # Send locale environment variables
 SendEnv <%= @node['ssh-hardening']['ssh']['client']['send_env'].join(' ') %>
 <% end %>
+
+# Extra Configuration Options
```

Comment:
Maybe we should do here something like 

```
<% if @node['ssh-hardening']['ssh']['client']['extras'].empty? %>
# Extra Configuration Options
<%- @node['ssh-hardening']['ssh']['client']['extras'].each do |key, value| %>
<%= key %> <%= value %>
<% end -%>
<% end -%>
```

This would allow us two tests:
 - Test if something configured - additional option will be added and '# Extra Configuration Options' is included in the file
 - Test if nothing is configured - there should be no '# Extra Configuration Options' in the file